packages/contracts/contracts/interfaces/v0.8.x/ISharedMinterDAV0.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface ISharedMinterDAV0 {
+    event SetAuctionDetails(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        uint256 _auctionTimestampStart,
+        uint256 _auctionTimestampEnd,
+        uint256 _startPrice,
+        uint256 _basePrice
+    );
```

Comment:
if we wanted to keep one event we could potentially just name the variable something that indicates it can be either/or, since they are the same type. but I am inclined to agree that we should just split as it's more readable and that is worth more here :)